lib/utils/transpile-es6.js

```diff
@@ -21,7 +21,7 @@ module.exports = function(tree, description, options) {
     transpiler = esperanto.toAmd;
   }
 
-  var outputTree = map(tree, '**/*.js', function(content, relativePath) {
+  var outputTree = map(tree, '**/*.!(umd).js', function(content, relativePath) {
```

Comment:
Unsure, I always have to play with it on the command line.  You could always just leave the glob, but `if (relativePath.match(/umd\.js$/)) { return '' }` inside the callback...